app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkItemMenuTest.kt

```diff
@@ -50,14 +50,25 @@ class BookmarkItemMenuTest {
     @Test
     fun `edit item appears for folders`() {
```

Comment:
I see these tests failing after the last patch because of
```
io.mockk.MockKException: no answer found for: PlacesBookmarksStorage(#11).getTree
```

Seems like this can be overcome by using something like
```
    fun `delete item has special styling`() = runBlocking {
        var deleteItem: TextMenuCandidate? = null
        mockkStatic("org.mozilla.fenix.ext.BookmarkNodeKt") {
            every { any<Context>().bookmarkStorage } returns mockk(relaxed = true)

            deleteItem = menu.menuItems(BookmarkNodeType.SEPARATOR, "").last()
        }
```
later mocking even the `getTree` response.

(`menuItems` sounds like a property to me, seems a bit strange to receive arguments but will leave it up to you if you think it would also sound better and offer more context as maybe `createMenuItems()`)